setup.py

```diff
@@ -201,6 +201,7 @@ def read_authors(filename):
             'analysis_level3_sameYear = bgcval2.analysis_level3_sameYear:main',
             'analysis_p2p = bgcval2.analysis_p2p:run',
             'analysis_timeseries = bgcval2.analysis_timeseries:main',
+            'parrallel_timeseries = bgcval2.parrellel_timeseries:main',
```

Comment:
I'd name it `batch_timeseries` -> it's not exactly parallel as in compute parallel (eg with MPI or multiprocessing/Pool)